wire-library/wire-moshi-adapter/src/main/java/com/squareup/wire/IdentityIfAbsent.kt

```diff
@@ -0,0 +1,11 @@
+package com.squareup.wire
```

Comment:
copyright

Reply:
hmm I noticed our `internal annotation class Uint64` doesn't have it either. Not needed for them?

Reply:
I like it for everything.

Reply:
Fixed x 2.